src/components/NavigationItems.module.scss

```diff
@@ -80,7 +80,6 @@ button.navLink {
   }
 }
 
-.isCurrentPage,
```

Comment:
Is there another definition for this class? Or does removing this mean there is no longer a need for this class. I see its still being used in the `.js` file so I'm wondering if we should remove that as well.

Reply:
there is yes, it's used to check that anything that is *not* isCurrentPage gets a hover style